### ISSUE TITLE
Issue when resuming training 

### ISSUE THREAD

**Issue by carbarey** [MaybeShewill-CV/bisenetv2-tensorflow#31]
Trying to resume training in Cityscapes of a Bisenetv2 from a checkpoint from previous training of around 100 epochs, the validation and training mIoUs drop to much lower levels than before, instead of continuing improving them. 

I've tried both disabling warm up and modifying learning rate manually in /config/cityscapes/cityscapes_bisenetv2.yaml, but the problem still occurs. Do you know why this behaviour occurs, and what could be made to fix it? Should some additional modification be done in the yaml file to resume training? Thanks!

**Comment by MaybeShewill-CV** [comment]
@carbarey Sounds like the model start to train from scratch rather than from your ckpt file:)

**Comment by carbarey** [comment]
Thanks for your answer! But I don't think that's the case, since the validation mIoU doesn't goes back to zero as would occur if training from scratch. Here is an example showing the validation mIoUs of saved checkpoints, when resuming from one already having a 0.4795 

/bisenetv2_11/cityscapes_val_miou=0.4795.ckpt-90

these are the first generated checkpoints - I always find this behavior where there is a small increase first (as expected) but then followed by a big jump where the mIoU decrease greatly, and then takes a long time to recover back to previous values:

/bisenetv2_12/cityscapes_val_miou=0.5145.ckpt-100.meta
/bisenetv2_12/cityscapes_val_miou=0.5143.ckpt-110.meta
/bisenetv2_12/cityscapes_val_miou=0.4088.ckpt-120.meta
/bisenetv2_12/cityscapes_val_miou=0.4336.ckpt-130.meta
/bisenetv2_12/cityscapes_val_miou=0.4503.ckpt-140.meta
/bisenetv2_12/cityscapes_val_miou=0.4626.ckpt-150.meta
/bisenetv2_12/cityscapes_val_miou=0.4772.ckpt-160.meta
/bisenetv2_12/cityscapes_val_miou=0.4890.ckpt-170.meta
/bisenetv2_12/cityscapes_val_miou=0.4991.ckpt-180.meta
/bisenetv2_12/cityscapes_val_miou=0.5100.ckpt-190.meta

I attach here my [cityscapes_bisenetv2.yaml](https://github.com/MaybeShewill-CV/bisenetv2-tensorflow/files/5583062/cityscapes_bisenetv2.yaml.txt) file (renamed to txt)

**Comment by MaybeShewill-CV** [comment]
@carbarey The sudden increase did happened in my experiments too. I can not figure out what happened to. But still you may use the eval script to eval the final model on test dataset when the training procedure totally complete. You may find the final miou was nothing different from normal trainning version:)
